clarity_ext/automated_tests/test_cases/dilute_file_reader/hamilton_driver_file_tests.py

```diff
@@ -0,0 +1,116 @@
+# Tests for reading from a Hamilton driver file,
+# using the utility dilute_filer_reader
+
+import unittest
+import os
+import inspect
+from ...utility.dilute_filer_reader.hamilton_driver_file_reader import HamiltonReader as FileReader
```

Comment:
We used relative paths before in the arteria project and figured that they are not recommended by the community (or the core team, don't remember the details). This can be fixed when we update the test structure.